images/hail-gcp-notebook/Dockerfile

```diff
@@ -11,7 +11,9 @@ RUN apt-get update && \
         openjdk-8-jre-headless \
         python3-pip && \
     rm -rf /var/lib/apt/lists/* && \
-    pip3 install --no-cache-dir \
+    conda create -n python310 python=3.10 ipykernel && \
```

Comment:
The notebook is based on conda environments, i.e. it'll display those as a kernel option to launch a notebook in. Otherwise I'd stay clear from it as well ;).